xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -390,8 +390,6 @@ namespace XBMCAddon
             item->GetVideoInfoTag()->m_strProductionCode = value;
           else if (key == "aired")
             item->GetVideoInfoTag()->m_firstAired.SetFromDateString(value);
-          else if (key == "credits")
-            item->GetVideoInfoTag()->m_writingCredits = StringUtils::Split(value, g_advancedSettings.m_videoItemSeparator);
```

Comment:
Because it was impossible to keep bw compatibility in that case in order to get a more consistent API. It was quite nonsense that getProperty() returned a window prop for xmbcgui.Window, but a container prop for xbmcgui.WindowXML (which inherits from xbmcgui.Window)

And sure, we could do nothing, and just keep stuff in our API which is good for anything but confusing devs. Not sure if that is a good idea though.